Sources/SwiftParser/generated/Parser+TokenSpecSet.swift

```diff
@@ -2084,3 +2084,30 @@ extension VariableDeclSyntax {
     }
   }
 }
+
+extension VersionTupleSyntax {
```

Comment:
This file is automatically generated and if you can’t manually add content to it. If we need this, we should place it in TokenSpecSet.swift

Reply:
Oh, I'm sorry. I missed that it was a subfile of `generated` 😅